Refresh token limitations (#4527)
postmanlabs/postman-docs: src/pages/docs/sending-requests/authorization.md

To manually refresh a token, select **Refresh** next to the token expiration time.

> > Note that this feature is available only when manually sending the request and functionality has not been extended for scheduled runs or monitors on the same collection.

Reviewer: ```suggestion
> Auto-refresh is only available when manually sending the request, and not for scheduled runs or monitors on the same collection.
```
